src/index.html

```diff
@@ -8,6 +8,58 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0 container" >
+      <div class="star---image"></div>
```

Comment:
According to name convention, use __ or - or -- to split words. Don't use --- in any place

Reply:
Read the checklist again